src/dashboard/src/components/api/views.py

```diff
@@ -582,6 +565,8 @@ def path_metadata(request):
         return helpers.json_response(body, status_code=201)
 
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET'])
 def processing_configuration(request, name):
```

Comment:
Maybe the return value could be sensitive to the request's content-type. I see that storage_service/locations/models/pipeline.py expects the response body to be an XML string, so that would have to change if the default return value is JSON (containing the XML in an object attribute, I assume)